contrib/cirrus/setup.sh

```diff
@@ -17,4 +17,26 @@ trap "complete_setup" EXIT
 msg "************************************************************"
 msg "Setting up runtime environment"
 msg "************************************************************"
+
 show_env_vars
+
+if [[ "$CIRRUS_TASK_NAME" == "build_cross" ]]; then
+	msg "Installing podman"
+	dnf -y install podman
```

Comment:
I'm pretty sure `podman` is already installed in the `c4801636756357120` image.

Reply:
@cevich uh I ran it without `podman` first and `cross` was unable to detect any container engine. But `dnf` should skip if its already there i guess.

Reply:
hmmm....odd....maybe there's a newer image with it.  It's def. in the package list in `automation_images`...

Reply:
Let's do this: https://github.com/containers/automation_images/pull/116